page-objects/pages/teamsPages/TMSAddEditTeamPage.js

```diff
@@ -22,27 +52,28 @@ async function setTeamDescription(description) {
 	await waitHelpers.waitAndSetValue(teamDescription, description);
 }
 
-async function selectColor() {
+async function selectColor(color) {
```

Comment:
I know it's not your implementation :) but it can be done by the way.  setColor

Reply:
done